src/utils/validators.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Validates an email address using a regular expression.
+ * @param {string} email - The email address to validate.
+ * @returns {boolean} Returns true if the email address is valid, false otherwise.
+ */
+export const validateEmail = (email: string): boolean => {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  return emailRegex.test(email);
+};
+/**
```

Comment:
const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
export const emailSchema = z.object({
  email: z
    .string()
    .min(1, { message: 'Email is required' })
    .regex(emailRegex, { message: 'Invalid email format' }),
});